packages/contracts/test/integration-tests/strategies/hack-money-vault.ts

```diff
@@ -11,30 +7,28 @@ import {
 } from "@lyrafinance/protocol/dist/scripts/util/web3utils";
 import {
   BasicFeeCounter__factory,
-  ERC20,
+  ERC20__factory,
 } from "@lyrafinance/protocol/dist/typechain-types";
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 import { expect } from "chai";
-import { Contract } from "ethers";
 import { ethers, network } from "hardhat";
-import { DEPLOYED_CONTRACTS } from "../../../constants";
 import { HackMoneyVault__factory } from "../../../typechain-types";
 import {
   HackMoneyStrategyLibraryAddresses,
   HackMoneyStrategy__factory,
 } from "../../../typechain-types/factories/HackMoneyStrategy__factory";
 import { HackMoneyStrategyDetailStruct } from "../../../typechain-types/HackMoneyStrategy";
+import { targets as lyraGlobal } from "./lyra-mainnet.json";
+
+const lyraMarket = lyraGlobal.markets.sETH;
 
 const strategyDetail: HackMoneyStrategyDetailStruct = {
-  maxVolVariance: toBN("0.1"),
-  gwavPeriod: 600,
   minTimeToExpiry: lyraConstants.DAY_SEC,
   maxTimeToExpiry: lyraConstants.WEEK_SEC * 2,
   mintargetDelta: toBN("0.15"),
   maxtargetDelta: toBN("0.85"),
-  maxDeltaGap: toBN("0.05"), // accept delta from 0.10~0.20 or 0.80~0.90
+  // maxDeltaGap: toBN("0.05"), // accept delta from 0.10~0.20 or 0.80~0.90
```

Comment:
why have you removed this @Baraa42 ?

Reply:
@igorline we don't use it in the strategy, and you asked to remove redundant vars